cluster.go

```diff
@@ -138,3 +145,62 @@ func (c *Cluster) GetLastReport() (*WorkloadReport, error) {
 	}
 	return &reports[0], nil
 }
+
+func (c *Cluster) getMetric(query string, t time.Time) (float64, error) {
```

Comment:
do we need some tests for them?

Reply:
We can mock it in the future.

Reply:
I think it may be difficult to mock promethus

Reply:
For br and prometheus, it seems like we need a ci first...